mutating-admission-controller/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang
+
+COPY . /usr/src/mutating-admission-controller
+
+WORKDIR /usr/src/mutating-admission-controller
```

Comment:
I think this will allow to de-duplicate the path and things should still work.
```suggestion
WORKDIR /usr/src/mutating-admission-controller

COPY . .
```